### ISSUE TITLE
Accessing Ubuntu filesystem from R-Instat on Wine

### ISSUE THREAD

**Issue by isedwards** [isedwards/r-instat#2]
Within R-Instat on Wine, is it possible to access the Ubuntu files and folders? When I go to `File` > `Open from File` it seems to just show Wine's Windows environment. Users will need to import their own data that's in the Ubuntu file system. Is that possible to set up for Wine?

<img width="960" alt="VirtualBox_Ubuntu Desktop 18 04 minimal_15_07_2020_09_53_24" src="https://user-images.githubusercontent.com/2097922/88453199-bf4c8e80-ce5c-11ea-831a-8791ba52e6bb.png">

**Comment by dannyparsons** [comment]
Apparently the z: drive is actually the Linux home directory https://askubuntu.com/a/32339 ? I'll confirm when I test again.

**Comment by dannyparsons** [comment]
Yes that seems to work. z/home/ubuntu is the location of the home directory in Linux. As the link above suggests, we might want to expose the home directory directly as h: to make it even easier for users if we can do that easily.
